### DISCUSSION TITLE
Unexpected Poor Results from Playground 2.5 with a Sci-Fi Cityscape Prompt

### DISCUSSION THREAD

**Discussion by realmalikov** [lllyasviel/Fooocus#3225] [Q&A]
I've been experimenting with Playground 2.5 recently, and I've generally been impressed with the quality of its outputs. However, I encountered a surprising issue with one particular prompt that yielded unexpectedly poor results. Do you have any tips for generating futuristic sci-fi cityscapes without the use of LoRas?

The Prompt:
"Futuristic cityscape from an epic sci-fi movie. Massive holographic advertisements floating between towering skyscrapers. Flying vehicles weaving through the buildings. Neon-lit streets below teeming with diverse alien species. Inspired by Blade Runner and The Fifth Element. Wide-angle shot, anamorphic lens flares, cyan and magenta color grading."

Issues Encountered:
1. Initially generated a 5:3 image with poor quality.
2. Even a 1:1 aspect ratio image produced highly disappointing results compared to SDXL models.

Observations:
1. The prompt seems clear and straightforward, making the poor output seem odd.
2. Typically, longer prompts haven't been problematic for the model.
3. The model appears to have no/bad understanding of some sci-fi concepts like futuristic skyscrapers and flying cars; simpler prompts like "Sci-fi movie still of an alien in a futuristic, mystic city" produce quite good results.

**Comment by DavidDragonsage** [comment]
I ran your prompt with Playground 2.5 out of curiosity. I received some very dense images, full of details and neon colours, tending towards sensory overload. I think the images were quite effective in that respect. There were lots of futuristic cars, but none flying.

I tried SDXLFaetastic_v24 with the same prompt. The images were more open, with discrete skyscraper buildings, also very futuristic. The streets were narrower and much less crowded. Futuristic cars again, but none flying.

Lastly, I ran the prompt with RealvisXLv40_v40Bakedvae. For me this was the ideal between the density of Playground and the discrete buildings of Faetastic. The buildings looked truly futuristic, with a lot of unique structures. There were a few vehicles on the surface, and LOTS of people. Hard to tell if they were alien. And there actually were some flying cars.

I have been using Fooocus Playground almost exclusively for the past two weeks and I am impressed by it in many respects. But I have also noticed odd shortcomings with certain prompts, and inconsistent quality with portraits - some were great and some needed lots of Inpainting (and Playground is useless for Inpainting). Rendering of human ethnicity is hit or miss - RealvisXLv40 is much more reliable in that respect.

And despite the Playground claims, in my experience the rendering of hands is no better than SDXL models.

So my approach is that if Playground produces good images for certain artistic themes, that's great - otherwise I go back to my favourite SDXL based models and don't sweat it.

My impression is that Playground is an innovative model but an immature one that still needs lots of work. The lack of LoRA support means that it has to standalone as it is, for better or worse.

**Reply by ILikeToasters** [reply]
Well that pretty much covers what I was going to say. I also think that with playground you need to prompt differently. Not sure the longer prompts do as well. As for inpainting try changing the inpaint engine in the debug settings to none. Not perfect but better.

**Reply by DavidDragonsage** [reply]
Quite right about setting the Inpaint engine to none. I was thinking about the Inpaint Detailer (Method #2), where the engine is always off. I got some pretty strange results with it. Now I just switch to a different model (preset) for any Inpainting.

**Reply by mashb1t** [reply]
also see https://github.com/lllyasviel/Fooocus/discussions/3217#discussioncomment-9929233

**Reply by DavidDragonsage** [reply]
Yes, "the new preset key default_inpaint_method will find its way to the playground preset" is a great idea!
